src/types.ts

```diff
@@ -67,7 +67,7 @@ export interface IDevWorkspace {
         routingClass: string;
         template: {
             projects?: any;
-            components?: any;
```

Comment:
Is it a good idea to start using types from devfile/api?

Reply:
Yeah, I'm going to bring those in in a future PR. I'll create an issue for it

Reply:
I've just created https://github.com/eclipse/che/issues/19579